helm/templates/metric-anomaly-detector/application-config.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.hypertraceAlertEngine.enabled}}
```

Comment:
This is for a single module vs an individual module, right?
So, if `.Values.hypertraceAlertEngine.enabled` is true means a single deployment mode that runs both task generator and evaluator, right?